package.json

```diff
@@ -67,7 +69,7 @@
     "husky": "4.3.0",
     "lint-staged": "10.3.0",
     "prettier": "2.1.1",
-    "semantic-release": "17.1.1",
+    "semantic-release": "^17.3.0",
```

Comment:
this one is fixing a high vulnerability @rbardini

Reply:
Cool, just remember to use exact versions here as well 🙂